iothub/device/src/Authentication/Security/StringValidationHelper.cs

```diff
@@ -24,7 +25,7 @@ public static void EnsureBase64String(string value, string paramName)
         {
             if (!IsBase64StringValid(value))
             {
-                throw new ArgumentException(CommonResources.GetString(Resources.StringIsNotBase64, value), paramName);
+                throw new ArgumentException(string.Format(CultureInfo.InvariantCulture, StringIsNotBase64, value), paramName);
```

Comment:
What is being formatted into the string "StringIsNotBase64"? I don't see a format placeholder (i.e., {0}).